packages/create/src/recipes/express/README.md

```diff
@@ -0,0 +1,51 @@
+# Express Integration Recipe for EdgeDB
+
+This recipe provides a starting point for building an [Express](https://expressjs.com/) application with EdgeDB as the database.
+
+We try to actively monitor and incorporate significant changes from the original "create-app" templates to to ensure developers have access to the latest features and best practices.
```

Comment:
FWIW, in this specific case, we aren't using any existing "create-app" as the starting point since I was unable to find anything that felt like a canonical starter for express.

Reply:
Ok, makes sense to remove it then